pkg/virt-operator/resource/generate/components/prometheus.go

```diff
@@ -553,6 +553,18 @@ func NewPrometheusRuleSpec(ns string, workloadUpdatesEnabled bool) *v1.Prometheu
 					vmStuckInStatusRule("starting"),
 					vmStuckInStatusRule("migrating"),
 					vmStuckInStatusRule("error"),
+					{
+						Alert: "KubeVirtNoAvailableNodesToRunVMs",
+						Expr:  intstr.FromString("((kube_node_labels{label_kubevirt_io_schedulable='true'} * on (node) kube_node_status_allocatable{resource='devices_kubevirt_io_kvm'}) or on() vector(0)) <= 0"),
```

Comment:
So there might not be kvm if the cluster would be intended for nesting. While we don't recommend this, could you reason why you want to include the kvm availability in the alert?

Reply:
KVM availability is the best way to know if hardware virtualization is found on a node. But taking into account the possibility to emulate, do you think it's best to remove the KVM check or add a new one for the emulation support configurations?

Reply:
@machadovilaca Indeed I think you should check that VMs can't run in any way, even with emulation (although I am not sure if something prevents us to run in emulated mode). @xpivarc Can you please elaborate on nesting?

Reply:
@enp0s3 @xpivarc 
I added a new metric to reflect the emulation configuration and added it to the alert expression